### PR TITLE
Upgrade to AWS SDK for Ruby - Version 2

### PR DESCRIPTION
**Pull request by lazaronixon**

### CODE REVIEW

File: spec/refile/spec_helper.rb [hunk 1]
```diff
@@ -61,6 +61,10 @@ def read(*args)
       @io.read(*args)
     end
 
+    def rewind
+      @io.rewind
+    end
```
Comment (jnicklas) [inline spec/refile/spec_helper.rb]: Why is this addition necessary?

Comment (lazaronixon) [inline spec/refile/spec_helper.rb]: https://github.com/aws/aws-sdk-ruby/blob/0d054b6eaaf6653210562e242a8586226565cbf4/aws-sdk-core/lib/seahorse/client/param_validator.rb#L106
